### DISCUSSION TITLE
[Nuxt 3] Layout title template problem

### DISCUSSION THREAD

**Discussion by svaldivia12** [nuxt/nuxt#17097] [Questions] [answered]
Hello! After updating my project packages, I've encounter an error in my page titles (before upgrading it was working fine). I'm using a title template to use this syntax: PageName | CompanyName
Now I'm getting the company name part duplicated in every page, something like this: PageName | CompanyName | CompanyName
My question is how can I use this title syntax without duplicating the last part.
Any help will be appreciated!

Code + Screenshots + Working example:

`pages/index.vue`
```
<script setup lang="ts">
console.log('-> Start index page');
useHead({
  title: 'IndexTitle',
});
console.log('<- End index page');
console.log('\n');
console.log('-----------------');
console.log('\n');
</script>

<template>
  <main>INDEX_PAGE</main>
</template>
```

`layouts/default.vue`
```
<script setup lang="ts">
console.log('-> Start layout');
useHead({
  titleTemplate: (page_title) => {
    const site_title = 'CompanyName';
    const final_title = page_title
      ? `${page_title} | ${site_title}`
      : `${site_title}`;
    console.log('page_title:', page_title);
    console.log('final_title:', final_title);
    console.log('\n');
    return final_title;
  },
});
console.log('<- End layout');
console.log('\n');
console.log('-----------------');
console.log('\n');
</script>

<template>
  <section>
    <slot />
  </section>
</template>
```

Title Screenshot:
![2022-08-10, 19-15-57  Screenshot](https://user-images.githubusercontent.com/104772973/184039683-d2a80a6f-cb4d-4e84-b094-89f6b4eb626c.png)

Browser Console:
![2022-08-10, 18-35-54  Screenshot](https://user-images.githubusercontent.com/104772973/184039755-1fc408b8-2846-4a75-b15f-8bcbc358c859.png)

Minimum example replication:
https://stackblitz.com/edit/nuxt-starter-zvkldu

**Comment by danielroe** [comment, marked as answer]
There has been a change in vueuse/head which now implements this instead of Nuxt.

You can pin your vueuse/head version to the previous one if you need to use rc6, or use the edge channel where it should be working fine.

**Reply by svaldivia12** [reply]
Thanks for your answer!

I just found the related issue, so I will leave it here to complement the answer.
https://github.com/nuxt/framework/issues/6294

Problem solved.

**Reply by svaldivia12** [reply]
I just realized that [RC7](https://github.com/nuxt/framework/pull/6375) has been released today, so I upgraded my packages, but the problem still persist.

Do I need to change something to get it to work properly again?

**Reply by danielroe** [reply]
If you are still experiencing an issue, then please do file a report (but probably it will be an issue with vueuse/head rather than Nuxt).
